db/models/score/scores.py

```diff
@@ -0,0 +1,40 @@
+"""The module containing all code related to the `scores` table
+within the Postgres db.
+"""
+import uuid
+from db import db
+from sqlalchemy import ForeignKey 
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class JustScore(db.Model):
+    """Score The sqlalchemy-flask model class used to define the
+    `scores` table in the Postgres database."""
+
+    __tablename__ = "just_scores"
+
+    just_score_id = db.Column(
+        "score_id", UUID(as_uuid=True), default=uuid.uuid4, primary_key=True
+    )
+
+    score = db.Column(
+        "score", db.Integer(), nullable=False
+    )
+
+    justification = db.Column(
+        "justification", db.Text(), nullable=False
+    )
+
+    application_id = db.Column(
+        "application_id", UUID, ForeignKey("assessment_records.application_id")
+    )
+
+    timestamp = db.Column(
+        "timestamp", db.DateTime(), nullable=False
+    )
+
+    sub_criteria_id = db.Column(
+        "sub_criteria_id", db.String(),  nullable=False
```

Comment:
According to the modal on Mural this should be a UUID rather than a String. I'm not sure where the details about the subcriteria are held though?

Reply:
CC @Adam-W1 does it make more sense for this to be a string, given the config dict?

Reply:
@Adam-W1 @gidsg yes! i was also confused by this, looking at the config dict i thought the most sensible thing to do was to keep it as a string otherwise sqlalchemy would complain about it not being a properly formed UUID 🤔

Reply:
If you use `UUID(as_uuid=True)` it might not complain anymore.

Reply:
@gio-karunakaran did you manage to fix this?